src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -736,32 +736,28 @@ private static ICalibratorTrainer GetCalibratorTrainerOrThrow(IExceptionContext
         /// </remarks>
         /// <param name="catalog">The multiclass classification catalog trainer object.</param>
         /// <param name="binaryEstimator">An instance of a binary <see cref="ITrainerEstimator{TTransformer, TPredictor}"/> used as the base trainer.</param>
-        /// <param name="calibrator">The calibrator. If a calibrator is not explicitly provided, it will default to <see cref="PlattCalibratorTrainer"/></param>
         /// <param name="labelColumnName">The name of the label column.</param>
         /// <param name="imputeMissingLabelsAsNegative">Whether to treat missing labels as having negative labels, instead of keeping them missing.</param>
-        /// <param name="maximumCalibrationExampleCount">Number of instances to train the calibrator.</param>
-        /// <param name="useProbabilities">Use probabilities (vs. raw outputs) to identify top-score category.</param>
+        /// <param name="useProbabilities">Use probabilities (vs. raw outputs) to identify top-score category. If specified, the type of the model must support probabilities or a runtime error will be thrown.</param>
```

Comment:
It seems like the default should be changed from `true` to `false`. Having a default that throws an exception doesn't seem a like a great beahvior IMO.

Reply:
I haven't run the benchmarks myself, though it seems like a large drop in metrics if you use uncalibrated models.

So the options would be:
* Default to `false` -- and give poor metrics (which makes the OVA rather useless)
* Keep default as `true` -- and throw an error if an uncalibrated model is used (forcing the user down a path to success)

Granted, similar to wschin's [comment](https://github.com/dotnet/machinelearning/pull/3928#discussion_r298361180), I wasn't expecting a very large drop in metric by using as uncalibrated models as the sort order of the base model's scores should be roughly the same.